geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -3395,11 +3399,10 @@ private InternalDistributedMember getNodeForBucketReadOrLoad(int bucketId) {
    * @throws PrimaryBucketException if the remote bucket was not the primary
    * @throws ForceReattemptException if the peer is no longer available
    */
+
   public boolean putRemotely(final DistributedMember recipient, final EntryEventImpl event,
       boolean ifNew, boolean ifOld, Object expectedOldValue, boolean requireOldValue)
       throws PrimaryBucketException, ForceReattemptException {
-    // boolean forceAck = basicGetWriter() != null
-    // || getDistributionAdvisor().adviseNetWrite().size() > 0;
```

Comment:
restore this comment; basically get this PR to the point that it does not modify PartitionedRegion.java

Reply:
updated